docs/building/windows-core.md

```diff
@@ -85,7 +85,7 @@ You can run our cross-platform Pester tests with `Start-PSPester`.
 Building in Visual Studio
 -----------------------------
 
-We do not recommend building the Poweshell solution from Visual Studio. This may lead to package version mismatches with errors similar to:
+We do not recommend building the Powershell solution from Visual Studio. This may lead to package version mismatches with errors similar to:
```

Comment:
As long as you are modifying it, please use "PowerShell" <-- Capitalize the S to match the project name